src/AppBundle/Resources/translations/common.en.yml

```diff
@@ -169,3 +169,10 @@ canYouAnswerNDR: |
   it's complete.
 returnToReport: Return to report overview
 returnToClientProfile: Return to client profile
+
+# Confirm delete page
+deletePage:
+    htmlTitle: Deputy report - remove %subject% | GOV.UK
+    question: Are you sure you want to remove this %subject%?
+    pageTitle: Remove %subject%
+    linkButtonLabel: Yes, remove %subject%
```

Comment:
These could be better off in a dedicated translation file?

Reply:
👍 Done